lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -50,6 +56,9 @@ def check_time_node(klass, node)
           return if (chain & DANGER_METHODS).empty? ||
                     !(chain & good_methods).empty?
 
+          return check_localtime(node) if (style == :acceptable) &&
```

Comment:
I dislike the use of modifier `if` when the condition doesn't fit on a single line. Change this to a normal `if` or create a helper method.